sci-biology/psychopy/psychopy-1.83.03.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit eutils gnome2-utils distutils-r1 git-r3
+
+DESCRIPTION="Python experimental psychology toolkit"
+HOMEPAGE="http://www.psychopy.org/"
+SRC_URI=""
+EGIT_REPO_URI="https://github.com/psychopy/psychopy.git"
+EGIT_COMMIT="1.83.03"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
```

Comment:
How do I pull a snapshot from a tag? Apparently there is an archive with the release at this address: https://github.com/psychopy/psychopy/releases/download/1.83.03/PsychoPy-1.83.03.zip Can I link to that with egit? or should I use SRC_URI?